receipt/src/test/java/com/hyperwallet/android/ui/receipt/viewmodel/ReceiptDetailViewModelTest.java

```diff
@@ -0,0 +1,49 @@
+package com.hyperwallet.android.ui.receipt.viewmodel;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import static com.hyperwallet.android.ui.receipt.view.ReceiptDetailActivity.EXTRA_RECEIPT;
+
+import android.content.Intent;
+
+import androidx.lifecycle.ViewModelProviders;
+
+import com.hyperwallet.android.exception.HyperwalletException;
+import com.hyperwallet.android.model.paging.HyperwalletPageList;
+import com.hyperwallet.android.model.receipt.Receipt;
+import com.hyperwallet.android.rule.HyperwalletExternalResourceManager;
+import com.hyperwallet.android.ui.receipt.view.ReceiptDetailActivity;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+
+@RunWith(RobolectricTestRunner.class)
+public class ReceiptDetailViewModelTest {
+
+    @Rule
+    public HyperwalletExternalResourceManager mExternalResourceManager = new HyperwalletExternalResourceManager();
+
+    @Test
+    public void testReceiptDetailViewModelInstance_isInitializedProperly() throws JSONException, HyperwalletException {
+
+        String json = mExternalResourceManager.getResourceContent("prepaid_card_receipt_list_response.json");
```

Comment:
I think we need to validate the following test cases:
1- create an instance of the ReceiptDetailViewModel, invoke the getReceipt and validate that it is null. Maybe name it testReceiptDetailViewModel_verifyDefaultValues
2- create an instance of the ReceiptDetailViewModel, invoke the setReceipt and make sure the receipt is set properly. Maybe name it testReceiptDetailViewModel_verifyReceiptSet or something like that